packages/venia-concept/src/components/Checkout/braintreeDropin.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * @fileoverview This component uses the BrainTree Web Drop In to
+ * hook into Web Payments and the Payment Request API to
+ * submit payments via BrainTree.
+ *
+ * See:
+ * https://github.com/braintree/braintree-web-drop-in
+ * https://braintree.github.io/braintree-web-drop-in/docs/current/index.html
+ * https://developers.braintreepayments.com/guides/drop-in/overview/javascript/v3
+ */
+
+import React, { Component } from 'react';
+import { bool, func } from 'prop-types';
+import dropin from 'braintree-web-drop-in';
+
+const { BRAINTREE_TOKEN } = process.env;
```

Comment:
This one's okay because `process.env` gets replaced with an object that has `BRAINTREE_TOKEN` as a key:

https://github.com/magento-research/pwa-studio/blob/464691ac5d9f2a247e5deb9c3d5413c3bf5a8648/packages/venia-concept/webpack.config.js#L121-L135